winservices.yml.sample

```diff
@@ -2,35 +2,44 @@ integrations:
   - name: com.newrelic.winservices
     exec:
       - C:\Program Files\New Relic\newrelic-infra\newrelic-integrations\bin\nri-winservices.exe
-      # exporter_bind_address and exporter_bind_port must be configured before running this integration
-      # The IP address and port used by the Prometheus Exporter to bind the web server.
+
+      # IP address and port used by the Prometheus exporter to bind the server.
+      #
       # - --exporter_bind_address
       # - 127.0.0.1
       # - --exporter_bind_port
       # - 9182
       
       # Time between consecutive metric collection of the integration.
       # It must be a number followed by a time unit (s, m or h), without spaces.
+      #
       - --scrape_interval
       - 30s
+
+      # By default, no service is included: to include services, create a list
+      # of filters to be applied to the service names. Filters are applied by
+      # order of precedence (within one level of precedence, the last matching
+      # pattern decides the outcome).
+      #
+      # Currently, only windowsService.name is supported for filtering.
+      # Prepend "!" to the pattern to exclude services.
+      # Prepend "regex" to indicate that the pattern is a regular expression.
+      #
+      # - --filter_list
+      # - |
+      #   windowsService.name:
+      #   regex ".*"
```

Comment:
Is it common to add `regex` keywork in case it is a regex in these use case?

Reply:
yes that is the usecase